app/src/main/java/com/usepace/android/messagingcenter/screens/sendbird/SendBirdChatFragment.java

```diff
@@ -988,23 +1044,27 @@ public void onSent(FileMessage fileMessage, SendBirdException e) {
     }
 
     private void editMessage(final BaseMessage message, String editedMessage) {
```

Comment:
You made changes here although the sdk doesn't support this feature yet

Reply:
did not change anything, might be spaces etc..